hack/gencerts.sh

```diff
@@ -17,6 +17,61 @@
 # Generates a CA certificate, a server key, and a server certificate signed by the CA.
 
 set -e
+SCRIPT=`basename ${BASH_SOURCE[0]}`
+
+function usage {
+  cat<< EOF
+  Usage: $SCRIPT
+  Options:
+  -h | --help               	   Display help information.
+  -n | --namespace <namespace>   The namespace where the Spark operator is installed.			
+  -p | --in-pod                  Whether the script is running inside a pod or not.
+EOF
+}
+
+function parse_arguments {
+  while [ $# -gt 0 ]
+  do
+    case "$1" in
+      -n|--namespace)
+      if [ -n "$2" ]; then
+        NAMESPACE="$2"
+      else
+        echo "-n or --namespace requires a value."
+        exit 1
+      fi
+      shift 2
+      continue
+      ;;
+      -p|--pod-container) # Whether the script is running inside a pod container or invoked manually
```

Comment:
Excellent work! One thing from me, do we really need this comment, since there is an explanation above? Please change it to `--in-pod`

Reply:
Good point. Yeah I forgot to change it here. Now fixed.